### ISSUE TITLE
Extensions descriptions: Keyboard focus is not visible on the "scroll up" button.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#88756]
**Environment Details:​**
VSCode Version: 1.42.0-insider (System setup)
OS Version: Win10

**Steps to Reproduce:**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Navigate  right panel till "Extension: Visual Studio Online" 
4. Check the keyboard focus is not visible on the "scroll up" button.

**Actual:**
Keyboard focus is not visible on the "scroll up" button.

**Expected:​**
Keyboard focus should  visible on the "scroll up" button.

**​User Impact:​**
Keyboard only user does not understand in which control keyboard focus present.
![image](https://user-images.githubusercontent.com/55909580/72524834-c466ca80-3888-11ea-85ad-5c5116491f9f.png)

**Comment by isidorn** [comment]
This feedback makes sense, that the extension description view scroll button shows focus feedback when it has focus.
First forwarding to @sandy081

**Comment by isidorn** [comment]
Closing in favor of https://github.com/microsoft/vscode/issues/88792
